osbs/cli/main.py

```diff
@@ -499,6 +517,17 @@ def cli():
                                  help="don't stop when restoring a resource fails")
     restore_builder.set_defaults(func=cmd_restore)
 
+    oauth2_builder = subparsers.add_parser(str_on_2_unicode_on_3('print-oauth-token-url'),
```

Comment:
We ought to have some consistency about how we refer to tokens. Here it's 'oauth-token', elsewhere it's 'oauth2-token' and 'token'.

Should we just use 'token' throughout?  e.g. osbs --token ..., "token = " in osbs.conf, --print-token-url, get-serviceaccount-token (as now).